doc/api/punycode.md

```diff
@@ -51,8 +63,14 @@ punycode.toUnicode('xn----dqo34k.com'); // '☃-⌘.com'
 ```
 
 ## punycode.ucs2
+<!-- YAML
+added: v0.6.12
```

Comment:
@bnoordhuis You are right. I'll fix it right away. But I'm a little confused why `punycode.ucs2` is not in nodejs v0.6.12.

punycode is updated to v1.0.0 in nodejs v0.6.12:
![image](https://cloud.githubusercontent.com/assets/171705/15368642/71cc047a-1d61-11e6-9d85-7af1b3204209.png)

<https://github.com/bestiejs/punycode.js/> had changed `punycode.utf16` to `punycode.ucs2` since v0.2.2:
![image](https://cloud.githubusercontent.com/assets/171705/15368809/5792cbce-1d62-11e6-83fe-6d8c5c0e8ae5.png)